src/main/java/io/jenkins/plugins/artifact_manager_s3/S3BlobStoreConfig.java

```diff
@@ -134,17 +134,17 @@ public ListBoxModel doFillRegionItems() {
     public FormValidation doCheckContainer(@QueryParameter String container){
         FormValidation ret = FormValidation.ok();
         if (StringUtils.isBlank(container)){
-            ret = FormValidation.error("The container name cannot be empty");
+            ret = FormValidation.warning("The container name cannot be empty");
         } else if (!bucketPattern.matcher(container).matches()){
-            ret = FormValidation.error("The container name does not match with S3 bucket rules");
+            ret = FormValidation.error("The S3 Bucket name does not match with S3 bucket rules");
         }
         return ret;
     }
 
     public FormValidation doCheckPrefix(@QueryParameter String prefix){
         FormValidation ret = FormValidation.ok();
         if (StringUtils.isBlank(prefix)){
-            ret = FormValidation.error("Prefix should have a value");
+            ret = FormValidation.warning("Prefix is needed to archive files in a subdirectory.");
         } else if (!prefix.endsWith("/")){
             ret = FormValidation.warning("if Prefix point to a folder, it should end with '/' character");
```

Comment:
This should be an `error` I think.

Reply:
but, we were talking about to allow empty prefix, I am lost.

Reply:
This comment is on the clause for checking for `/` when the prefix is _not_ empty.

IOW I am suggesting something along the lines of

```java
if (StringUtils.isBlank(prefix)) {
    return FormValidation.ok("BTW you might want a prefix etc.");
} else if (prefix.endsWith("/")) {
    return FormValidation.ok();
} else {
    return FormValidation.error("A prefix must end with a slash.");
}
```